fpmsyncd/routesync.cpp

```diff
@@ -747,6 +754,79 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
 }
 
 /* 
+ * Handle label route
+ * @arg nlmsg_type      Netlink message type
+ * @arg obj             Netlink object
+ */
+void RouteSync::onLabelRouteMsg(int nlmsg_type, struct nl_object *obj)
```

Comment:
if "obj" doesn't change make it const.

Reply:
Not possible.  libnl3 does not use "const" for the "struct nl_object*" in its function definitions, so obj cannot be const when passed to libnl3 functions in onLabelRouteMsg.  This is the reason nl_object is never declared const in routesync.h evn though it does not change.